dannysanchez559/react-challenge-project-sep-trial
# Ds 8 route guarding

## Changes
_List Changes Introduced by this PR_
1. add implementation for route guarding

## Purpose
https://github.com/Shift3/react-challenge-project-sep-trial/issues/6

## Testing Steps
from the login form, simply append "order-form" or "view-orders" to the url on the address bar. 
If not logged in, the user should not be able to access those pages and should simply stay on the login form. 

_If this closes an issue, name it here. If it doesn't, remove this line_
Closes #8
